mathics/builtin/box/graphics3d.py

```diff
@@ -388,6 +388,7 @@ def boxes_to_json(self, leaves=None, **options):
                 },
                 "lighting": self.lighting,
                 "viewpoint": self.viewpoint,
+                "revision": "1.0",
```

Comment:
I thought we were going to use "protocol" ?

Reply:
My typo, I've written protocol everywhere else.